### PR TITLE
Create first part of the git lesson

### PR DESCRIPTION
**Pull request by aguspesce**

### CODE REVIEW

File: 03_git_part_1.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
+![CoCo banner](./media/coco-banner.jpg)
+
+# Introduction to Git and GitHub | Introducción a Git y GitHub
+
+In this opportunity we will learn the basics of what Git and Github are.
```
Comment (mdpiper) [inline 03_git_part_1.md:5]: Perhaps reword to  _In this lesson we will learn the basics of Git and Github._

File: 03_git_part_1.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+- Work with remote repositories (GitHub) | Trabajar con repositorios remotos
+  (GitHub)
+
+## Contents | Contenidos
+
+- What is Automated Version Control? | ¿Que es el Control Automatizado de
+  Versiones?
+- Git configuration | Configuración Git
+- How to create a repository | Cómo crear un repositorio
+- How to register the changes | Cómo registrar los cambios
```
Comment (mdpiper) [inline 03_git_part_1.md:34]: Reword to _How to register changes_

File: 03_git_part_1.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+- Remote repositories on GitHub | Repositorios remotos en GitHub
+
+## Automated Version Control | Control Automatizado de Versiones
+
+The version control can be used to keep track of what one person did and when.
```
Comment (mdpiper) [inline 03_git_part_1.md:39]: Perhaps reword to: _Version control keeps track of what a person did and when._

File: 03_git_part_1.md [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+
+---
+
+El control de versiones se puede utilizar para realizar un seguimiento de lo
+que hizo una persona y cuándo.
+
+Los sistemas de control de versiones nos permiten hacer lo mismo que en la
+historieta pero de forma mas eficiente, limpia y ordenada.
+
+<img src="data/git/phdcomics.png" alt="" style="height: 10vh">
```
Comment (mdpiper) [inline 03_git_part_1.md:52]: Would you like to make this bigger?

**Comment by aguspesce** [comment]
@mdpiper ...Can you see the new text?
I added the homework and the last section about remote repositories

**Review by mdpiper — APPROVED** [review]

File: 03_git_part_1.md [hunk 5]
```diff
@@ -0,0 +1,468 @@
...
+<!-- files in the current directory (and all directories below it) to their state at -->
+<!-- the commit specified. This command will restore data_cruncher.py to the latest -->
+<!-- commit version, but it will also restore any other files that are changed to -->
+<!-- that version, erasing any changes you may have made to those files! As -->
+<!-- discussed above, you are left in a detached HEAD state, and you don’t want to -->
+<!-- be there.  -->
+
+### Remote repository in GitHub | Repositorio remoto en GitHub
+
+GitHub allows us to storage copies of our repositories to shear them with
```
[the remark this answers is not in the document]
Comment (aguspesce) [reply 03_git_part_1.md]: Done! Thank you!

**Review by mdpiper — APPROVED** [review]

**Pull request merged**